wire/message.go

```diff
@@ -159,7 +159,9 @@ func makeEmptyMessage(command string) (Message, error) {
 		msg = &MsgCFilterV2{}
 
 	default:
-		return nil, fmt.Errorf("unhandled command [%s]", command)
+		const op = "makeEmptyMessage"
```

Comment:
Please define this at the very top of the func for consistency with the rest.  The reasoning is that it makes it easier to spot if it doesn't match the function name.